os/storage/using-zfs/index.md

```diff
@@ -94,7 +93,8 @@ At this point you'll have a ZFS filesystem created and mounted at `/zpool1/docke
 Now you'll need to remove `-s overlay` (or any other storage driver) from the Docker daemon args to allow docker to automatically detect `zfs`.
```

Comment:
I don't know what you mean here, because the following code snippet does exactly what is described in this sentence: removes `-s overlay` from docker daemon arguments.